wikipedia-sentences/README.md

```diff
@@ -52,28 +61,6 @@ shuf input.txt > input.txt
 
 To shuffle a file on macos, please read [this post](https://apple.stackexchange.com/questions/142860/install-shuf-on-os-x/195387).
 
-## 🔍 Search
```

Comment:
Any reason why we are not supporting REST or box?

Reply:
They are supported and included now in the latest readme